### PR TITLE
fix a couple of label css

### PR DESCRIPTION
**Pull request by akash1810**
# search
## before
label-before

## after
label

# gr-panel
## before
screen shot 2015-09-21 at 14 32 00

## after
screen shot 2015-09-21 at 14 30 18

upload remains as it is.

### CODE REVIEW

File: kahuna/public/js/edits/labeller.css [hunk 1]
```diff
@@ -17,13 +19,18 @@
     background-color: #99ddff;
 }
 
-.label--partial,
-.label--partial gr-icon {
+/*
+label--partial is part of gr-panel.
+TODO refactor gr-panel to use common labeller component
+*/
+.label--partial .label__value,
```
Comment (jamesgorrie) [inline kahuna/public/js/edits/labeller.css]: Could we not just apply this to `label--partial`? Why have every building block have it?

Comment (akash1810) [inline kahuna/public/js/edits/labeller.css]: To win on specificity against https://github.com/guardian/grid/pull/1282/files#diff-493a18df09d616f3d63820529e14407cR33 ... I agree its not very nice.

Comment (jamesgorrie) [inline kahuna/public/js/edits/labeller.css]: Ah, crapola - this works for now though - let's tidy it up once this is out.

**Comment by jamesgorrie** [comment]
:+1:

**Pull request merged**